### ISSUE TITLE
 Is iCUE SDKv4 supported?

### ISSUE THREAD

**Issue by AxisRay** [DarthAffe/RGB.NET#293]
the example doesn't work on my computer.

Device: Corsair Dominator DDR5 Memory
iCUE Version: v4.33.138

```
using RGB.NET.Core;
using RGB.NET.Devices.Corsair;
using RGB.NET.Devices.Asus;
using RGB.NET.Presets.Decorators;
using RGB.NET.Presets.Textures;
using RGB.NET.Presets.Textures.Gradients;

Console.WriteLine("Hello, World!");

RGBSurface surface = new RGBSurface();
surface.Load(CorsairDeviceProvider.Instance);
surface.AlignDevices();

surface.RegisterUpdateTrigger(new TimerUpdateTrigger());

ILedGroup allLeds = new ListLedGroup(surface, surface.Leds);
RainbowGradient rainbow = new RainbowGradient();
rainbow.AddDecorator(new MoveGradientDecorator(surface));
ITexture texture = new ConicalGradientTexture(new Size(10, 10), rainbow);
allLeds.Brush = new TextureBrush(texture);

Console.ReadLine();
```

**Comment by DarthAffe** [comment]
For now you need to use the latest 3. sdk ([CUE SDK v3.0.464](https://github.com/CorsairOfficial/cue-sdk/releases/tag/v3.0.464))
I'm working on integration the v4 but it's not done yet.

**Comment by DarthAffe** [comment]
Support for the iCUE-SDK v4 is merged (#297) and part of the latest pre-release nuget.
